supabase/schemas/04_sessions.sql

```diff
@@ -2,6 +2,7 @@ create table sessions (
   id uuid primary key default gen_random_uuid(),
   track_id uuid references tracks(id) on delete cascade,
   organization_member_id uuid references organization_members(id) on delete cascade,
+  comment_id integer default null,
```

Comment:
I think storing the comment_url here would make more sense than storing the comment_id. What do you think?

Reply:
I agree about that 👍